indico/modules/events/client/js/reviewing/components/ChangesConfirmation.jsx

```diff
@@ -0,0 +1,66 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import {Form as FinalForm} from 'react-final-form';
+import {Form, Message, Divider} from 'semantic-ui-react';
+import {FinalTextArea} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+import './ChangesConfirmation.module.scss';
+
+export default function ChangesConfirmation() {
+  const confirmChanges = formData => {
+    console.log(formData);
+  };
+
+  return (
+    <div>
+      <Divider />
+      <FinalForm onSubmit={confirmChanges}>
+        {fprops => (
+          <Form id="confirmation-form" onSubmit={fprops.handleSubmit}>
+            <Message
+              info
+              size="tiny"
+              icon="exclamation circle"
+              header={Translate.string('Confirmation required')}
+              content={Translate.string(
+                'Editor uploaded changes to your paper. Please download and confirm. You can \
+                  reject the changes and provide a new version by yourself.'
+              )}
+            />
+            <FinalTextArea
+              name="comment"
+              placeholder={Translate.string('If you wish you can leave a comment...')}
+            />
+            <Form.Group inline styleName="submit-buttons">
+              <Form.Button
```

Comment:
Use the logic from `FinalSubmitButton` to disable those buttons while submitting etc.

Best option is probably to pass along extra props in the FinalSubmitButton component so you can use it here.